quickstart/sample-apps/k3s-clustered/Makefile

```diff
@@ -0,0 +1,46 @@
+include ../Makefile
+VM_NAME := <INTEGRATION_SLUG>-sample-app
+VM_CLUSTER_MAIN := $(VM_NAME)-k3s-main
+CONFIG_FILE_DIR := jinja/variables
+CONFIG_FILE_VARS := $(CONFIG_FILE_DIR)/monitoring-config.yaml
+CONFIG_OUT_FILE := monitoring-config.yaml
+TEMPLATE_FILE_DIR := jinja/templates
+TEMPLATE_FILE := $(TEMPLATE_FILE_DIR)/monitoring-config-template.yaml
+RIVER_FILE := configs/suo.river
+LOKI_INSTANCE := your-loki-instance:3100
+PROMETHEUS_INSTANCE := your-prometheus-instance:9090
+
+.PHONY: run-ci stop install-suo default-monitoring-config render-config install-monitoring transfer-configs clean
+
+run-ci: clean k3s-setup-cluster install-suo default-monitoring-config render-config install-monitoring
+
+stop: k3s-stop-cluster clean
+
+install-suo: transfer-configs
+	@multipass transfer ./scripts/suo_setup.sh $(VM_CLUSTER_MAIN):/home/ubuntu/suo_setup.sh;
+	@multipass exec $(VM_CLUSTER_MAIN) -- bash -c "chmod +x suo_setup.sh && ./suo_setup.sh"
+
+render-config:
+	@docker run --rm -v $(shell pwd)/$(TEMPLATE_FILE_DIR):/templates -v $(shell pwd)/$(CONFIG_FILE_DIR):/variables dinutac/jinja2docker:latest /templates/monitoring-config-template.yaml /variables/monitoring-config.yaml --format=yaml > monitoring-config.yaml
```

Comment:
I'm a bit worried that we use :latest images in this repo as it may lead to some unexpected breakdowns in the future. Thoughts? Should we pin some versions instead?

Reply:
Yeah you're right, for sample-apps it makes sense to pin versions. Crossed my mind when working on the mongodb sample app, but never did it properly. Specifically for `apt` based installs as well would be useful, along with calling out the multipass (ubuntu) codename explicitly, rather than it being inferred and meaning latest.

Will add a backlog task to go through and pin them